src/collective/fingerpointing/upgrades/configure.zcml

```diff
@@ -0,0 +1,29 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:five="http://namespaces.zope.org/five"
+    xmlns:i18n="http://namespaces.zope.org/i18n"
+    xmlns:gs="http://namespaces.zope.org/genericsetup">
+
+  <include file="profiles.zcml" />
+
+  <gs:registerProfile
```

Comment:
instead of registering this profile with duplicate content on it, let's read from the default profile and apply those steps with Python code